Mathlib/CategoryTheory/Sums/Basic.lean

```diff
@@ -187,6 +187,34 @@ def sum (F : A ⥤ B) (G : C ⥤ D) : Sum A C ⥤ Sum B D
     | inr X, inr Y, inr Z, f, g => by erw [G.map_comp]; rfl
 #align category_theory.functor.sum CategoryTheory.Functor.sum
 
+/-- Similar to `sum`, but both functors land in the same category `C` -/
+def sum' (F : A ⥤ C) (G : B ⥤ C) : Sum A B ⥤ C
+    where
+  obj X :=
+    match X with
+    | inl X => F.obj X
+    | inr X => G.obj X
+  map := @fun X Y f =>
```

Comment:
The following syntax is better. (Do not hesitate doing the same change in all the file.)
```suggestion
  map {X Y} f :=
```

Reply:
Indeed, this syntax looks neater. I just changed all such cases in the file uniformly.